projects/plugins/jetpack/extensions/blocks/videopress/resumable-upload/index.js

```diff
@@ -106,6 +106,39 @@ export default function ResumableUpload( { file } ) {
 
 	const fileSizeLabel = filesize( file.size );
 
+	const getErrorMessage = () => {
+		let errorMessage = __(
+			'An error was encountered during the upload. Check your network connection.',
+			'jetpack'
+		);
+		if ( typeof error === 'object' ) {
+			const apiResponse = error.toString().match( /message":"([^"]+)"/ );
+			// tus doesnt give us direct acces to the API response, but let's try to parse it to provide useful feedback for the user.
+			if ( typeof apiResponse === 'object' && apiResponse.length === 2 ) {
+				const apiResponseMessage = apiResponse[ 1 ];
+				// Let's give this error a better message.
+				if ( apiResponseMessage === 'Invalid Mime' ) {
+					errorMessage = (
+						<>
+							{ __( 'The format of the video you uploaded is not supported.', 'jetpack' ) }
+							&nbsp;
+							<a
+								href="https://wordpress.com/support/videopress/recommended-video-settings/"
```

Comment:
Should we be using `getRedirectUrl()` here for that link?

Reply:
See https://github.com/Automattic/jetpack/pull/23376

TL;DR is we are not doing it for blocks yet because we don't want simple site users to be redirected to `jetpack.com` and then back to `wordpress.com`.